manual/applicant.md

```diff
@@ -0,0 +1,95 @@
+# Applying for access rights
+
+Applicants can apply for access to different kinds of resources using REMS (Resource Entitlement Management System).
+
+You do not need to create a user account to log in to the service because REMS uses external identity providers, such as Haka identity and Suomi.fi e-Identification. Use the identity provider your organisation/institution recommends.
+
+:exclamation: Note! Remember to always use the same login because your applications are connected to the login identity you use.
+
+## How to apply for access rights
+
+You can find all the resources you can apply for access from the Catalogue tab when you are logged in. The Catalogue tab also shows your unfinished applications. Continue filling in an application by selecting **View**.
+
+1. Start the application process by adding one or more resources to your shopping cart.
+If the resources have the same application form, they will be bundled together automatically. This means that you can apply for access rights to several resources with just one application form.
```

Comment:
I guess this should be updated to also talk about the same workflow now that we can support multiple forms in one application.

Reply:
Does it make any difference for the applicant where the form items come from?

Reply:
I think for an applicant the more correct phrasing would be just "If many items can be applied for together with one application, they will be automatically bundled together for you. Otherwise you can always do multiple applications."

Reply:
Corrected.